core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1138,7 +1165,7 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
           val validMemberFuture = removeMembersResult.memberResult(new MemberToRemove(testInstanceId))
           assertNull(validMemberFuture.get())
 
-          // The group should contain no member now.
+          // The group's active members number should decrease by 1
```

Comment:
We could remove this comment for now

Reply:
Removed, but curious about the reason :)